client/index.jsx

```diff
@@ -100,6 +103,7 @@ Promise.all([rootElemPromise])
     return { elem, store }
   })
   .then(async ({ elem, store }) => {
+    const configPromise = fetchJson('/config', { method: 'get' })
```

Comment:
imo this should be included in `_sbInitData` in the web version, we want to avoid as many extra requests as we can